src/darker/command_line.py

```diff
@@ -23,7 +24,7 @@ def parse_command_line(argv: List[str]) -> Namespace:
     parser = ArgumentParser(
         description="\n".join(description), formatter_class=NewlinePreservingFormatter,
     )
-    parser.add_argument("src", nargs="*")
+    parser.add_argument("src", nargs="+")
```

Comment:
With this, the description of the positional argument in `--help` and the error from an omitted source file path are
```
$ darker --help
                [...]
positional arguments:
  src
                [...]
$ darker
                [...]
darker: error: the following arguments are required: src
```

While we're improving the command line parser, maybe this would be a better:
```
$ darker --help
                [...]
positional arguments:
  PATH                  Path(s) to the Python source file(s) to reformat
                [...]
$ darker
                [...]
darker: error: following arguments are required: PATH
```
To get that, and a better description of the positional arguments in `--help`, we could do this:
```suggestion
    parser.add_argument(
        "src",
        nargs="+",
        help="Path(s) to the Python source file(s) to reformat",
        metavar="PATH",
    )
```